pkg/textparse/openmetricsparse_test.go

```diff
@@ -272,71 +272,91 @@ func TestOpenMetricsParseErrors(t *testing.T) {
 		err   string
 	}{
 		{
-			input: "",
+			input: "# EOF",
```

Comment:
This is valid input.

Reply:
Yes, the error is EOF; which means that it is valid. It was like it before.

Reply:
This needs a comment then.

Reply:
Added